src/ape/api/transactions.py

```diff
@@ -145,7 +145,7 @@ class ReceiptAPI(BaseInterfaceModel):
     a :class:`ape.contracts.base.ContractInstance`.
     """
 
-    contract_address: Optional[str] = None
+    contract_address: Optional[AddressType] = None
```

Comment:
Should sender and receiver also be `AddressType` ?

Reply:
Yes, but different PR